.changeset/late-suits-talk.md

```diff
@@ -0,0 +1,5 @@
+---
```

Comment:
Unless I'm missing something, I don't think this requires a changeset/release as the keywords are only used on the website.

Reply:
@Dhaulagiri, good call out. I didn't think of it. Just took it off